searchcore/src/vespa/searchcore/config/proton.def

```diff
@@ -446,7 +446,7 @@ initialize.threads int default = 0
 
 ## Portion of max address space used in components in attribute vectors
 ## before put and update operations in feed is blocked.
-writefilter.attribute.address_space_limit double default = 0.9
+writefilter.attribute.address_space_limit double default = 0.92
```

Comment:
Intentional to change address_space_limit to 0.92 ?

Reply:
Yes. The fail safe limit in proton should hit later than the limit used by the cluster controller.